rest-api/src/main/java/gov/cms/qpp/conversion/api/exceptions/NoFileInDatabaseException.java

```diff
@@ -0,0 +1,8 @@
+package gov.cms.qpp.conversion.api.exceptions;
+
+/**
+ * Exception for an database not
+ */
+public class NoFileInDatabaseException extends RuntimeException {
+	public NoFileInDatabaseException(String message) {super(message);}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public constructor by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)